### PR TITLE
Release v1 post

### PR DESCRIPTION
**Pull request by davidfischer**
- Backstory
- ML model link and details
- Future: More Niches and Sponsorship

## Questions

- Should we mention collaborating with the SpaCy devs?
- This talks about PSF sponsorship. Is that ok?

## Screenshot
Screenshot 2022-08-25 at 15-39-31 Going v1 0 The Backstory and Future of Our Ad Network

### CODE REVIEW

File: content/posts/2022-version-v1-0.md [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+but there wasn't any monetization.
+It was a service with millions of monthly visitors that thousands of projects relied on for their docs
+but run by folks with day jobs who wore pagers for their side project.
+It wasn't sustainable.
+
+The obvious way to monetize was ads.
+However, we cared about the privacy of our visitors
+and we didn't want ad companies to track our users around the web.
+We went to a few ad networks and asked if we could proxy ad traffic or even just run ads without cookies.
+They laughed at us and told us to take it or leave it.
```
Comment (ericholscher) [inline content/posts/2022-version-v1-0.md:30]: A bit over the top, but probably fine :)

File: content/posts/2022-version-v1-0.md [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+
+
+> "EthicalAds' targeting provides a better ad experience for our community members."
+>
+> &mdash; Ido Shamun, Co-founder & CTO of daily.dev
+
+
+Today, we now have ~130 publishers and we gross just under $60k per month in ad revenue
+of which 70% goes to publishers.
+Most of our publishers are small sites or open source projects so to send them ~$40k/mo feels great.
```
Comment (ericholscher) [inline content/posts/2022-version-v1-0.md:47]: Should this be a pull quote too? Or highlight the revenue a bit more in some fashion? It feels like a throwaway as currently written, but I think it's one of the key things.

File: content/posts/2022-version-v1-0.md [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+
+
+## How EthicalAds works
```
Comment (ericholscher) [inline content/posts/2022-version-v1-0.md:50]:
```suggestion
## Building a content-targeted ad network with machine learning
```

I think we're kind of burying the lede here with all the backstory. I think it's probably fine, but it really feels like "we have an ad network with $60k in revenue, and just shipped an awesome ML model as 1.0" is pretty hidden when I look at the overall flow of this content.

File: content/posts/2022-version-v1-0.md [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+or "Web Development with Go".
+
+However, as we get larger, our contextual targeting will scale well as our network scales
+and these highly focused campaigns will be possible.
+It's not an understatement to say that this kind of more advanced contextual targeting is the future of EthicalAds.
+
+
+> "EthicalAds made it simple to help promote all the PSF sponsors across our web properties, without worrying about user tracking."
+>
+> &mdash; Ee Durbin, Director of Infrastructure - Python Software Foundation
```
Comment (ericholscher) [inline content/posts/2022-version-v1-0.md:89]: Sadly we don't have a 👍 to use this. I think we probably just mention it.

```suggestion
> The Python Software Foundation is using EthicalAds to power sponsored messages across their web ecosystem.
>
> Check it our on the sidebar of the [Python Package Index](https://pypi.org/project/urllib3/1.26.7/#changes)
```

File: content/posts/2022-version-v1-0.md [hunk 5]
```diff
@@ -0,0 +1,101 @@
...
+
+Another area we are looking at is to help [sponsorships]({filename}/pages/landing-pages/sponsorship.md)
+at large organizations deliver more value for sponsors.
+Currently, we're helping the Python Software Foundation
+to promote their sponsors across their ecosystem of web properties
+while also allowing them to report important metrics like how many times a sponsor's logo was seen and clicked.
+
+
+We're really excited about implementing new features on our ad network and continuing to iterate on sponsorships,
```
Comment (ericholscher) [inline content/posts/2022-version-v1-0.md:98]:
```suggestion
We're really excited about implementing new features on our ad network and continuing to iterate on sponsorships.
```

File: content/posts/2022-version-v1-0.md [hunk 6]
```diff
@@ -0,0 +1,101 @@
...
+> "EthicalAds made it simple to help promote all the PSF sponsors across our web properties, without worrying about user tracking."
+>
+> &mdash; Ee Durbin, Director of Infrastructure - Python Software Foundation
+
+Another area we are looking at is to help [sponsorships]({filename}/pages/landing-pages/sponsorship.md)
+at large organizations deliver more value for sponsors.
+Currently, we're helping the Python Software Foundation
+to promote their sponsors across their ecosystem of web properties
+while also allowing them to report important metrics like how many times a sponsor's logo was seen and clicked.
+
```
Comment (ericholscher) [inline content/posts/2022-version-v1-0.md:96]:
```suggestion

# Join us for what's next
```

File: content/posts/2022-version-v1-0.md [hunk 7]
```diff
@@ -0,0 +1,101 @@
...
+Date: August 30, 2022
+description: After two years, we released v1.0 of our ad server. We wanted to share our story and our progress in building a privacy focused ad network for devs including more details of our machine learning model.
+tags: ad-server, community, machine-learning
+authors: David Fischer
+image: /images/posts/2022-version-1-0.jpg
+image_credit: <span>Photo by <a href="https://unsplash.com/es/@an_ku_sh?utm_source=unsplash&utm_medium=referral&utm_content=creditCopyText">Ankush Minda</a> on <a href="https://unsplash.com/s/photos/release?utm_source=unsplash&utm_medium=referral&utm_content=creditCopyText">Unsplash</a></span>
+
+
+A few weeks ago, we released the v1.0 version of our ad server.
+While versioning schemes are somewhat arbitrary,
```
Comment (ericholscher) [inline content/posts/2022-version-v1-0.md:11]: This is not what we want our second line to be. We should be hooking the reader, not getting philosophical about version numbers.

File: content/posts/2022-version-v1-0.md [hunk 8]
```diff
@@ -0,0 +1,101 @@
...
+releasing v1.0 felt like a significant milestone (v1.0.0 implies stability in [SemVer](https://semver.org/)).
```
Comment (ericholscher) [inline content/posts/2022-version-v1-0.md:12]: Why is it a milestone? We should mention why we made it 1.0 here!

**Review by ericholscher — APPROVED** [review]
This is a great post, but I do worry we're burying the lede a bit. I think we probably want a pull quote for our revenue & ML approaches, or some other way to highlight that it's the meat of the announcement, and perhaps a bit less of the backstory? We can always link to https://www.ethicalads.io/advertising-vision/ which covers it pretty well. 

This is fine to go out mostly as is, but I do think we'll get more traction if it has a bit more of a hook up front.

File: content/posts/2022-version-v1-0.md (hunk 3 above)
Comment (davidfischer) [reply content/posts/2022-version-v1-0.md:50]: I'm fine with the suggestion and definitely acknowledge that this is pretty far down in the post to talk about the ML model. However, I do think some amount of backstory is important and I think the backstory flows into the revenue numbers which are also important.

**Comment by davidfischer** [comment]
I pushed a few updates based on the feedback. Hopefully this makes the ML part more obvious.

**Comment by ericholscher** [comment]
Yea. Looks good. I still think we're talking a lot about history up top, but it's probably fine. 👍

File: content/posts/2022-version-v1-0.md (hunk 3 above)
Comment (ericholscher) [reply content/posts/2022-version-v1-0.md]: Yea, I don't know the right balance, but I think this is fine to ship it.

**Pull request merged**